src/h5web/visualizations/nexus/hooks.ts

```diff
@@ -18,6 +19,8 @@ import {
 import { getChildEntity } from '../../utils';
 
 export function useNxData(group: Group): NxData {
+  assertNxDataGroup(group);
```

Comment:
This is the assertion that replaces `findNxDataGroup` in the vis containers, as mentioned.